charts/apisix/values.yaml

```diff
@@ -211,7 +211,13 @@ etcd:
   service:
     port: 2379
 
-  replicaCount: 3
+  statefulset:
+    replicaCount: 3
```

Comment:
We need to make the configuration here consistent with bitnami/etcd https://github.com/bitnami/charts/blob/080ab2485dfe9ba060710aad18c2edb6381fa543/bitnami/etcd/values.yaml#L249-L251